library/src/blas3/rocblas_trmm.hpp

```diff
@@ -122,15 +139,17 @@ rocblas_status rocblas_trmm_template(rocblas_handle    handle,
     //
     // -- Rewritten for gemm based trmm for rocBLAS
     //
-    T zero = 0.0;
-    T one  = 1.0;
+    T  zero   = 0.0;
```

Comment:
Didn't go through the original trmm algorithm, just went through the changes and quickly went through to ensure the indexes were translated right, looks fine as long as tests pass.